.github/cspell.json

```diff
@@ -2,45 +2,186 @@
   "version": "0.2",
   "language": "en",
   "flagWords": [],
-  "words": [
-    "ASDW",
-    "backtick",
-    "backticks",
-    "blockquote",
-    "broadphase",
-    "conformant",
-    "dartdoc",
-    "dartdocs",
-    "dashbook",
-    "discoverability",
-    "fireslime",
-    "flippable",
-    "hitbox",
-    "linkcheck",
-    "LTRBR",
-    "LTWH",
-    "markdownlint",
-    "platformer",
-    "positionable",
-    "prerendered",
-    "pressable",
-    "proxying",
-    "raycast",
-    "raytracing",
-    "refactor",
-    "refactoring",
-    "refactorings",
-    "renderable",
-    "socio",
-    "Strikethrough",
-    "stylesheet",
-    "sublists",
-    "typedef",
-    "typedefs",
-    "toctree",
-    "toctrees",
-    "vsync"
-  ],
+  "words":['AHSL', 
```

Comment:
This is not propper JSON-formatting, it should be indented in the same way as it was previously.